jwst/extract_1d/extract_1d_step.py

```diff
@@ -412,20 +412,19 @@ def process(self, input):
                     self.log.info('Exposure is in the SUBSTRIP256 subarray.')
                     self.log.info('Traces 1 and 2 will be modelled and decontaminated before extraction.')
                     subarray = 'SUBSTRIP256'
-                elif input_model.meta.subarray.name == 'FULL':
-                    self.log.info('Exposure is in the FULL subarray.')
-                    self.log.info('Traces 1 and 2 will be modelled and decontaminated before extraction.')
-                    subarray = 'FULL'
                 elif input_model.meta.subarray.name == 'SUBSTRIP96':
                     self.log.info('Exposure is in the SUBSTRIP96 subarray.')
                     self.log.info('Traces of orders 1 and 2 will be modelled but only order 1'
                                   ' will be decontaminated before extraction.')
                     subarray = 'SUBSTRIP96'
                 else:
-                    self.log.error('The SOSS extraction is implemented for the SUBSTRIP256,'
-                                   ' SUBSTRIP96 and FULL subarray only.')
-                    self.log.error('extract_1d will be skipped.')
+                    self.log.error('The SOSS extraction is implemented for the SUBSTRIP256'
+                                   'and SUBSTRIP96 subarrays only. Subarray is currently '
+                                   f'{input_model.meta.subarray.name}.')
+                    self.log.error('Extract1dStep will be skipped.')
+                    self.log.error('PhotomStep will be skipped: spec not created in extract_1d')
                     input_model.meta.cal_step.extract_1d = 'SKIPPED'
+                    input_model.meta.cal_step.photom = 'SKIPPED'
```

Comment:
This makes me a bit uncomfortable - I don't think steps should be attempting set keywords for other steps, if at all possible. Will make a suggestion on how to do this in photom.